lxc/remote.go

```diff
@@ -326,6 +326,29 @@ func (c *cmdRemoteAdd) Run(cmd *cobra.Command, args []string) error {
 		}
 
 		conf.Remotes[server] = config.Remote{Addr: addr, Public: true, Protocol: c.flagProtocol}
+
+		// Check that the server is reachable before saving the config
+		d, err := conf.GetImageServer(server)
+		if err == nil {
+			_, err = d.GetImages()
+			if err != nil {
+				return err
+			}
+
+			return conf.SaveConfig(c.global.confPath)
+		}
+
+		// If the server is not reachable as image server, try to add it as an instance server
```

Comment:
why?

Reply:
Its been specified as a simplestreams server on line 323.

Reply:
Right, I realize this is useless.. removing it.